src/js-v2-api-specification.md

```diff
@@ -0,0 +1,309 @@
+| SERVICES       | Services Hub Page      |
+| YOUR_HEALTH    | Your Health Hub Page   |
+| MESSAGES       | Messages Hub Page      |
+| UPLIFT         | P9 Uplift Page         |
+
+
+##### Example Usage
+
+```js
+nhsapp.navigation.goToPage(nhsapp.navigation.AppPage.MESSAGES)
+```
+
+##### Status
+
+Live
+
+---
+
+#### openBrowserOverlay Function <a name="openBrowserOverlay"></a>
+
+`nhsapp.navigation.openBrowserOverlay(overlayUri: Uri)`
+
+##### Description
+
+Redirects the user to a specified page that will open in a browser overlay view.
+
+##### Parameters
+
+| Parameters              |                                                                                                                                                                |
+|-------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| overlayUri                 | **Type:** Uri *Required*<br>The page to open in the browser overlay.                                                                                                     |
+
+##### Example Usage
+
+```js
+nhsapp.navigation.openBrowserOverlay("https://www.nhs.uk/")
+```
+
+##### Status
+
+Live
+
+---
+
+#### openExternalBrowser Function <a name="openExternalBrowser"></a>
+
+`nhsapp.navigation.openExternalBrowser(browserUri: Uri)`
+
+##### Description
+
+Redirects the user to a specified page that will open in the device's default browser application.
+
+##### Parameters
+
+| Parameters              |                                                                                                                                                                |
+|-------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| browserUri              | **Type:** Uri *Required*<br>The page to open in the browser.                                                                                                     |
+
+##### Example Usage
+
+```js
+nhsapp.navigation.openExternalBrowser("https://www.nhs.uk/")
+```
+
+##### Status
+
+Live
+
+---
+
+### Storage <a name="storage"></a>
+
+#### addEventToCalendar Function <a name="addToCalendar"></a>
+
+```	javascript
+nhsapp.storage.addEventToCalendar(
+  subject: string,
+  body: string,
+  location: string,
+  startTimeEpochInSeconds: integer,
+  endTimeEpochInSeconds: integer,
+)
+```
+
+##### Description
+
+Add an event to the calendar on the user's device. The NHS App's native controls will alert the user of any issues adding the event to their calendar.
+
+#### Parameters
+
+| Parameters              |                                                                                                                                                                |
```

Comment:
Not sure how much it matters, but whitespace alignment.

Reply:
fixed